Src/loadelf.c

```diff
@@ -369,15 +369,16 @@ static void _processFunctionDie( struct symbol *p, Dwarf_Debug dbg, Dwarf_Die di
         attr_tag = DW_AT_abstract_origin;
         dwarf_attr( die, attr_tag, &attr_data, 0 );
         dwarf_global_formref( attr_data, &abstract_origin_offset, 0 );
-        dwarf_offdie_b( dbg, abstract_origin_offset, IS_INFO, &abstract_origin_die, 0 );
-        isinline = true;
-    }
-    else
-    {
-        dwarf_highpc_b ( die, &h, 0, &formclass, 0 );
-        dwarf_lowpc ( die, &l, 0 );
+        if (DW_DLV_OK == dwarf_offdie_b( dbg, abstract_origin_offset, IS_INFO, &abstract_origin_die, 0 ))
```

Comment:
There has been an issue with inline low/high addresses, where function names could not be matched to addresses this should be fixed with this part.